config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  # get "/users" to: "users#index"
```

Comment:
You did great job :). However, there is a tiny issue.

- [ ] Kindly remove unused commented code from your project.